tox.ini

```
@@ -0,0 +1,26 @@
+[tox]
+envlist = py3{8,9,10,11,12}, mypy, black, pylint
+isolated_build = True
+skip_missing_interpreters = True
+
+[testenv]
+deps = pytest
+commands = pytest
```

To make sure we're using the right interpreter, and to be able to pass pytest arguments like `tox -e py311 -- --stepwise`:
```suggestion
commands = {envpython} -m pytest  {posargs}
```

And maybe add `{envpython} -m` for the other envs?

---

How would it be possible for tox to pick "the wrong" pytest? It does now allows for executables outside the tox venv, itself created with the right version?

---

Ah right, yes, I guess we should be fine without `{envpython}` now.

`{posargs}` would be useful.

---

TIL about `{posargs}`, I'll add it, it looks nice!